api/lib/source/cosmosV0-source.js

```diff
@@ -288,6 +299,60 @@ class CosmosV0API extends RESTDataSource {
     )
   }
 
+  async getTopVoters() {
+    // for now defaulting to pick the 5 largest voting powers
+    return _.take(
+      _.reverse(
+        _.sortBy(this.store.validators, [
+          (validator) => {
+            return validator.votingPower
+          }
+        ])
+      ),
+      5
+    ).map(({ operatorAddress }) => operatorAddress)
```

Comment:
why return just the operatorAddress and not directly the whole validator object? this way you can directly display the validtor without needing to map in the FE

Reply:
because of what we already discussed. We need to agree on a new type, `topVoter`, that also deals with the council member case

Reply:
some council members are not validators

Reply:
I think this would be a different PR though

Reply:
OK, I'm going to investigate what data I can get from council members and create a `type topVoter`

Reply:
Created